src-docs/src/views/super_date_picker/super_date_picker_example.js

```diff
@@ -137,39 +149,183 @@ if (!endMoment || !endMoment.isValid()) {
+            <li>
+              <EuiCode>recentlyUsedRanges</EuiCode>: Store the users previously
+              submitted time ranges and pass them back to date picker with this
+              props. It&apos;s best to limit this list to around 10 items.
+            </li>
+            <li>
+              <EuiCode>customQuickSelectPanels</EuiCode>: Accepts an array of
+              panel objects as{' '}
+              <EuiCode language="tsx">
+                {'{ title: string, content: ReactElement }'}
+              </EuiCode>
+              .
+            </li>
+          </ul>
           <p>
-            <strong>EuiSuperDatePicker</strong>&apos;s quick select menu also
-            supports <strong>custom panels</strong>. These panels can have their
-            own title and perform custom actions on the date picker.
+            You can also reduce the <strong>EuiSuperDatePicker</strong> to
+            display <strong>just</strong> the quick select button and dropdown
+            by passing{' '}
+            <EuiCode language="tsx">{'isQuickSelectOnly={true}'}</EuiCode>. Be
+            sure you display the rendered time period elsewhere in your
+            interface.
           </p>
-        </div>
+        </>
       ),
       snippet: superDatePickerCustomQuickSelectSnippet,
       demo: <SuperDatePickerCustomQuickSelect />,
     },
+    {
+      title: 'Sizing',
+      source: [
+        {
+          type: GuideSectionTypes.TSX,
+          code: superDatePickerWidthSource,
+        },
+      ],
+      text: (
+        <>
+          <p>
+            By default the <EuiCode>width</EuiCode> of the{' '}
+            <strong>EuiSuperDatePicker</strong> is set to{' '}
+            <EuiCode>{"'restricted'"}</EuiCode> which sets the size to a
+            reasonable max-width necessary to display full start and end date
+            strings.
+          </p>
+          <p>
+            You can adjust the <EuiCode>width</EuiCode> by passing:
+          </p>
+          <ul>
+            <li>
+              <EuiCode>{"'full'"}</EuiCode> to expand the width to 100% of the
+              parent
+            </li>
+            <li>
+              <EuiCode>{"'auto'"}</EuiCode> to grow and shrink as the contents
+              does
+            </li>
+          </ul>
+          <p>
+            The <strong>EuiSuperDatePicker</strong> also supports the{' '}
+            <EuiCode>compressed</EuiCode> form control option.
+          </p>
+        </>
+      ),
+      demo: <SuperDatePickerWidth />,
+      props: { EuiSuperDatePicker },
+    },
+    {
+      title: 'Auto refresh',
+      source: [
+        {
+          type: GuideSectionTypes.TSX,
+          code: autoRefreshSource,
+        },
+      ],
+      text: (
+        <p>
+          By supplying a callback function to <EuiCode>onRefreshChange</EuiCode>
+          , the <strong>EuiSuperDatePicker</strong> will display the{' '}
+          <EuiCode>refreshInterval</EuiCode> configuration UI.
```

Comment:
Documentation suggestion - I was personally confused by this example and what the difference was between it and the preceding example because I'm a super noob and don't use the quick select panel normally.

```suggestion
          , the <strong>EuiSuperDatePicker</strong> will display the{' '}
          <EuiCode>refreshInterval</EuiCode> configuration UI in the <strong>Quick Select</strong> menu <EuiIcon type="calendar" color="primary" />.
```

[This probably needs to be spaced correctly]

Alternatively, maybe consider turning on the auto refresh interval by the default to make the demo more obvious?